versa/utterance_metrics/asvspoof_score.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Jiatong Shi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# Author: You (Neil) Zhang
+
+## This is to evalute the generated speech with a deepfake detection model
+# We include the AASIST model trained on ASVspoof 2019 LA dataset to 
+# output the confidence score of whether the speech input is a deepfake
+# Please refer to https://github.com/clovaai/aasist for more details
+
+
+## cloning the repo into the checkpoint folder
+## git clone git@github.com:clovaai/aasist checkpoints/aasist
```

Comment:
We may add an installer file to `tools` for this option?

Reply:
Updated.